lms/static/scripts/tsconfig.json

```diff
@@ -14,7 +14,6 @@
   },
   "include": ["**/*.js", "**/*.ts", "**/*.tsx"],
   "exclude": [
-    "new-application-instance.js",
```

Comment:
While checking this file, I noticed it references this file which doesn't exist any more.